tuning/sft_trainer.py

```diff
@@ -238,12 +246,20 @@ def train(
     }
 
     json_dataset = datasets.load_dataset("json", data_files=data_files)
-    formatted_train_dataset = json_dataset["train"].map(format_dataset)
+    if data_args.data_formatter_template:
+        formatted_train_dataset, data_args.dataset_text_field = \
+            formatting_function(json_dataset["train"], data_args.data_formatter_template, tokenizer.eos_token)
```

Comment:
IMO it's a little confusing to name the formatter for the custom templates `formatting_function`. I think the reason is that to me `formatting_function` sounds like something that you would map over the dataset, while `format_dataset` sounds like something that does the mapping, but in the `if/else` here, it's the opposite.

Any thoughts on renaming this `apply_custom_formatting_template` or something?

Reply:
changed name as suggested